zksync_sdk/wallet.py

```diff
@@ -255,6 +255,68 @@ async def transfer(self, to: str, amount: Decimal, token: TokenLike,
 
         return await self.send_signed_transaction(transfer, eth_signature)
 
+    async def get_order(self, token_sell: TokenLike, token_buy: TokenLike,
+                        ratio: Fraction, ratio_type: RatioType, amount: Decimal,
+                        recipient: str = None,
+                        valid_from=DEFAULT_VALID_FROM,
+                        valid_until=DEFAULT_VALID_UNTIL) -> Order:
+        account_id, nonce = await self.zk_provider.get_account_nonce(self.address())
+        token_sell = await self.resolve_token(token_sell)
+        token_buy = await self.resolve_token(token_buy)
+        recipient = recipient or self.address()
+
+        if ratio_type == RatioType.token:
```

Comment:
Looks weird, why we don't need to do this procedure for `Wei` and what does that mean? Is it for `Eth`?

Reply:
[Here is what this means.](https://zksync.io/dev/swaps.html#utils)

We don't need to do this for Wei because the server expects it to be in Wei. For Token, we just convert it to Wei.